diff --git a/man/Makefile.am b/man/Makefile.am
index 338c26c..c75f0dc 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -193,7 +193,7 @@ MANS_COMMON = \
 	manpage-common-legacy-output.man
 
 $(MANS_CPP): $(MANS_CPP:%=%.pre)
-	$(CPP) -w -C -P -I$(top_srcdir)/man $@.pre  $@
+	$(CPP) -nostdinc -w -C -P -I$(top_srcdir)/man $@.pre  $@
 
 CLEANFILES = $(MANS_CPP) *~
 DISTCLEANFILES = $(MANS_CPP) $(MANS_CPP:%=%.pre) .deps/*.P
