From: Ferenc Wágner <wferi@niif.hu>
Description: fix typos found by lintian
Origin: Debian
Forwarded:  will be
Last-Update: 2012-06-23
--- a/man/ipmi-oem.8.pre.in
+++ b/man/ipmi-oem.8.pre.in
@@ -119,7 +119,7 @@
 This OEM command will set the current SSH configuration on the IPMI
 card.  The possible keys and values are \fIssh=enable|disable\fR,
 \fIidletimeout=seconds\fR, and \fIportnumber=num\fR.  Multiple
-key=value pairs may be specified.  If no key=value pairs are specifed,
+key=value pairs may be specified.  If no key=value pairs are specified,
 available pairs are output.  Some fields may be read-only on specific
 Poweredge systems.  Command confirmed to work on Dell Poweredge R610
 and R710 (Dell 11G Poweredge systems).
@@ -134,7 +134,7 @@
 card.  The possible keys and values are \fItelnet=enable|disable\fR,
 \fIsessiontimeout=seconds\fR, \fIportnumber=num\fR, and
 \fI7fls=enable|disable\fR.  Multiple key=value pairs may be specified.
-If no key=value pairs are specifed, available pairs are output.  Some
+If no key=value pairs are specified, available pairs are output.  Some
 fields may be read-only on specific Poweredge systems.  Command
 confirmed to work on Dell Poweredge R610 and R710 (Dell 11G Poweredge
 systems).
@@ -149,7 +149,7 @@
 IPMI card.  The possible keys and values are
 \fIwebserver=enable|disable\fR, \fIsessiontimeout=seconds\fR,
 \fIhttpportnumber=num\fR, and \fIhttpsportnumber=num\fR.  Multiple
-key=value pairs may be specified.  If no key=value pairs are specifed,
+key=value pairs may be specified.  If no key=value pairs are specified,
 available pairs are output.  Some fields may be read-only on specific
 Poweredge systems.  Command confirmed to work on Dell Poweredge R610
 and R710 (Dell 11G Poweredge systems).
@@ -183,7 +183,7 @@
 \fIgcfilter3=string\fR,
 \}
 and \fIcertificatevalidation=enable|disable\fR.  If no key=value pairs
-are specifed, available pairs are output.  Some fields may be
+are specified, available pairs are output.  Some fields may be
 read-only on specific Poweredge systems.  Command confirmed to work on
 Dell Poweredge R610 and R710 (Dell 11G Poweredge systems).
 .TP
@@ -375,7 +375,7 @@
 \fIsubject=string\fR, \fImessagecontent=string\fR,
 \fIsenderemailaddress=string\fR, and \fIsmtphostname=string\fR.
 Multiple key=value pairs may be specified.  If no key=value pairs are
-specifed, available pairs are output.  Command confirmed to work on
+specified, available pairs are output.  Command confirmed to work on
 Intel S5500WB (Penguin Computing Relion 700) and Intel S2600JF (Appro 512X).
 .TP
 .B get-power-restore-delay 
@@ -515,7 +515,7 @@
 any of the settings will disable the lockout feature.  Each time any
 of these settings is modified, the authentication failure count of
 each enabled user is reset to 0.  Multiple key=value pairs may be
-specified.  If no key=value pairs are specifed, available pairs are
+specified.  If no key=value pairs are specified, available pairs are
 output.  Command confirmed to work on Inventec 5441/5442 (Dell Xanadu
 II/III).
 .TP
@@ -549,7 +549,7 @@
 should be assigned from the DHCP server.  \fIdnsdomainname\fR
 specifies the DNS domain name string.  This field is read only if
 \fIdnsdomainnamedhcp\fR is enabled.  Multiple key=value pairs may be
-specified.  If no key=value pairs are specifed, available pairs are
+specified.  If no key=value pairs are specified, available pairs are
 output.  Command confirmed to work on Inventec 5441/5442 (Dell Xanadu
 II/III).
 .TP
@@ -563,7 +563,7 @@
 IPMI card.  The possible keys and values are
 \fIwebserver=enable|disable\fR, \fIwebservertimeout=seconds\fR,
 \fIhttpportnumber=num\fR, and \fIhttpsportnumber=num\fR.  Multiple
-key=value pairs may be specified.  If no key=value pairs are specifed,
+key=value pairs may be specified.  If no key=value pairs are specified,
 available pairs are output.  Command confirmed to work on Inventec
 5441/5442 (Dell Xanadu II/III).
 .TP
@@ -584,7 +584,7 @@
 minimum and maximum configured, and \fIuser\fR uses the user defined
 time defined by \fIpowerondelay\fR.  \fIpowerondelay\fR must be within
 the minimum and maximum power on delay times.  Multiple key=value
-pairs may be specified.  If no key=value pairs are specifed, available
+pairs may be specified.  If no key=value pairs are specified, available
 pairs are output.  Command confirmed to work on Inventec 5441/5442
 (Dell Xanadu II/III).
 .TP
@@ -784,7 +784,7 @@
 should be assigned from the DHCP server.  \fIdnsdomainname\fR
 specifies the DNS domain name string.  This field is read only if
 \fIdnsdomainnamedhcp\fR is enabled.  Multiple key=value pairs may be
-specified.  If no key=value pairs are specifed, available pairs are
+specified.  If no key=value pairs are specified, available pairs are
 output.  Command confirmed to work on Quanta S99Q (Dell FS12-TY).
 .TP
 .B get-web-server-config
@@ -796,7 +796,7 @@
 IPMI card.  The possible keys and values are
 \fIwebserver=enable|disable\fR, \fIwebservertimeout=seconds\fR,
 \fIhttpportnumber=num\fR, and \fIhttpsportnumber=num\fR.  Multiple
-key=value pairs may be specified.  If no key=value pairs are specifed,
+key=value pairs may be specified.  If no key=value pairs are specified,
 available pairs are output.  Command confirmed to work on Quanta S99Q
 (Dell FS12-TY).
 .TP
@@ -817,7 +817,7 @@
 minimum and maximum configured, and \fIuser\fR uses the user defined
 time defined by \fIpowerondelay\fR.  \fIpowerondelay\fR must be within
 the minimum and maximum power on delay times.  Multiple key=value
-pairs may be specified.  If no key=value pairs are specifed, available
+pairs may be specified.  If no key=value pairs are specified, available
 pairs are output.  Command confirmed to work on Quanta S99Q (Dell
 FS12-TY).
 .TP
--- a/man/ipmi-sensors.8.pre.in
+++ b/man/ipmi-sensors.8.pre.in
@@ -25,7 +25,7 @@
 by a BMC.  Sensors may output a sensor event of "Unknown" if the
 sensor reading cannot be read.  This is typical of a sensor that is
 busy or a reading that cannot be calculated.  If sensors report
-"Unrecognized State", it is indicative of an unkown sensor type,
+"Unrecognized State", it is indicative of an unknown sensor type,
 typically an OEM sensor.  If the sensor OEM interpretation is
 available, the \fB\-\-intepret\-oem\-data\fR may be able to report
 the appropriate sensor state.  Sensors need not always report a sensor
--- a/ipmi-chassis-config/ipmi-chassis-config-boot-flags.c
+++ b/ipmi-chassis-config/ipmi-chassis-config-boot-flags.c
@@ -1214,7 +1214,7 @@
     "boot off of.  Most users may wish to select NO-OVERRIDE to select the "
     "configuration currently determined by the BIOS.  Note that the configuration "
     "value BIOS-SETUP refers to booting *into* the BIOS Setup, not from it.  FLOPPY "
-    "may refer to any type of removeable media.  \"Device_Instance_Selector\" may "
+    "may refer to any type of removable media.  \"Device_Instance_Selector\" may "
     "be be used to select a specific device instance for booting.";
 
   assert (state_data);
--- a/libfreeipmi/sel-parse/ipmi-sel-parse-string-intel.c
+++ b/libfreeipmi/sel-parse/ipmi-sel-parse-string-intel.c
@@ -656,7 +656,7 @@
 	  else if (event_special_code == IPMI_SENSOR_TYPE_MEMORY_EVENT_DATA2_OEM_INTEL_QUANTA_QSSC_S4R_EVENT_SPECIAL_CODE_MEMORY_HOT_PLUG_TIMEOUT)
 	    event_special_code_str = "Memory Hot-plug Timeout";
 	  else if (event_special_code == IPMI_SENSOR_TYPE_MEMORY_EVENT_DATA2_OEM_INTEL_QUANTA_QSSC_S4R_EVENT_SPECIAL_CODE_USER_INITIATED_CANCELATION)
-	    event_special_code_str = "User-initiated cancelation";
+	    event_special_code_str = "User-initiated cancellation";
 	  else
 	    event_special_code_str = "Unknown";
 
--- a/libfreeipmi/include/freeipmi/spec/ipmi-comp-code-spec.h
+++ b/libfreeipmi/include/freeipmi/spec/ipmi-comp-code-spec.h
@@ -205,7 +205,7 @@
 #define IPMI_COMP_CODE_SEND_MESSAGE_INVALID_SESSION_HANDLE                                                            0x80
 #define IPMI_COMP_CODE_SEND_MESSAGE_INVALID_SESSION_HANDLE_STR \
   "Invalid Session Handle.  The session handle does not match up with any " \
-  "curently active sessions for this channel."
+  "currently active sessions for this channel."
 
 #define IPMI_COMP_CODE_SEND_MESSAGE_LOST_ARBITRATION                                                                  0x81
 #define IPMI_COMP_CODE_SEND_MESSAGE_LOST_ARBITRATION_STR \
